packages/graphql-language-service-server/src/MessageProcessor.js

```diff
@@ -453,8 +459,15 @@ function convertToRpcMessage(metaMessage: Object): ResponseMessage {
   return message;
 }
 
-function invalidateCache(textDocument: Object, content: Object): void {
-  const uri = textDocument.uri;
+function invalidateCache(
+  textDocument: Object,
+  uri: Uri,
+  content: Object,
+): void {
+  if (!uri) {
```

Comment:
We chatted offline but yeah, this effectively does nothing but to halt the execution without `Uri` to invalidate the cache with. I'll turn this into an error message later.